.toys/ci.rb

```diff
@@ -433,6 +437,12 @@ def encode_str str
   
   def create_body dir, tasks
     now = Time.now.utc.strftime "%Y-%m-%d %H:%M:%S"
-    "At #{now} UTC, detected failures in #{dir} for: #{tasks.join ', '}"
+    if github_action_id
```

Comment:
Currently, `github_action_id` is set to the empty string if the `--github-action-id` flag is omitted (see line 374 where you set the default to the empty string). So this if statement always succeeds (and is thus useless). However, that means the `action_url` will be set to a URL without an action ID if the `--github-action-id` flag is omitted, which is probably not what you want. Maybe consider removing the default (which will cause `github_action_id` to be nil if the flag is omitted) and then include appropriate logic to detect that condition and omit the "The CI logs can be found here" line in that case.

Reply:
Thanks for spotting this. I assumed empty string evaluates to false. 🤦‍♂️

I removed the default value and added a check for `nil` value.